build/.goreleaser.yml

```diff
@@ -0,0 +1,134 @@
+    binary: nri-prometheus
+    gobinary: "go1.13.15"
+    ldflags:
+      - -s -w -X internal/integration/integration.Version={{.Version}} #-X main.gitCommit={{.Commit}} -X main.buildDate={{.Date}}
+    goos:
+      - linux
+      - darwin
+      - freebsd
+    goarch:
+      - 386
+      - amd64
+      - arm
+      - arm64
+    ignore:
+      - goos: darwin
+        goarch: 386
+      - goos: freebsd
+        goarch: arm
+      - goos: freebsd
+        goarch: arm64
+
+  - id: nri-prometheus-win
+    main: ./cmd/nri-prometheus/
+    binary: nri-prometheus
+    gobinary: "go1.13.15"
+    ldflags:
+      - -s -w -X internal/integration/integration.Version={{.Version}} #-X main.gitCommit={{.Commit}} -X main.buildDate={{.Date}}
+    goos:
+      - windows
+    goarch:
+      - 386
+      - amd64
+    hooks:
+      pre: build/windows/set_exe_properties.sh {{ .Env.TAG }} "prometheus"
+
+nfpms:
+  - id: nri-prometheus-deb
+    builds:
+      - nri-prometheus-nix
+    file_name_template: "{{ .ProjectName }}_{{ .Version }}-1_{{ .Arch }}"
+    vendor: "New Relic, Inc."
+    homepage: "https://www.newrelic.com/infrastructure"
+    maintainer: "New Relic Infrastructure Team <infrastructure-eng@newrelic.com>"
+    description: "New Relic Infrastructure Prometheus Integration extends the core New Relic\nInfrastructure agent's capabilities to allow you to collect metric and\nlive state data from prometheus components."
+    license: "https://newrelic.com/terms (also see LICENSE installed with this package)"
+    # Formats to be generated.
+    formats:
+      - deb
+    bindir: "/var/db/newrelic-infra/newrelic-integrations/bin"
+    files:
+      # "prometheus-config.yml.sample": "/etc/newrelic-infra/integrations.d/prometheus-config.yml.sample"
+      "CHANGELOG.md": "/usr/share/doc/nri-prometheus/CHANGELOG.md"
+      "README.md":  "/usr/share/doc/nri-prometheus/README.md"
+      "LICENSE": "/usr/share/doc/nri-prometheus/LICENSE"
+    #config_files:
+    #  "prometheus-definition.yml": "/var/db/newrelic-infra/newrelic-integrations/prometheus-definition.yml"
+    dependencies:
+      - "newrelic-infra"
+
+  - id: nri-prometheus-rpm
+    builds:
+      - nri-prometheus-nix
+    file_name_template: "{{ .ProjectName }}-{{ .Version }}-{{ .Arch }}"
+    vendor: "New Relic, Inc."
+    homepage: "https://www.newrelic.com/infrastructure"
+    maintainer: "New Relic Infrastructure Team <infrastructure-eng@newrelic.com>"
+    description: "New Relic Infrastructure Prometheus Integration extends the core New Relic\nInfrastructure agent's capabilities to allow you to collect metric and\nlive state data from prometheus components."
+    license: "https://newrelic.com/terms (also see LICENSE installed with this package)"
+    # Formats to be generated.
+    formats:
+      - rpm
+    bindir: "/var/db/newrelic-infra/newrelic-integrations/bin"
+    files:
+      # "prometheus-config.yml.sample": "/etc/newrelic-infra/integrations.d/prometheus-config.yml.sample"
```

Comment:
I commented this out because we don't have them. We'll have to add them back when we have the config sample files